Keyboard.js

```diff
@@ -8,6 +8,10 @@ define([
 	'./util/misc',
 	'dojo/_base/sniff'
 ], function (declare, aspect, domClass, on, lang, has, miscUtil) {
+	
+	var hasPointer = has("pointer"),
+            hasMSPointer = hasPointer && hasPointer.slice(0, 2) === "MS",
```

Comment:
We also have https://github.com/dojo/dojo/commit/b25b080d6fb0e1b9e64ba48e303d207614f80ada ... so again, I think if we properly use has('msPointer') that will work fine for Dojo 1.10+, though we should probably include the same pointer logic for pre-Dojo 1.10 dgrid users...